docs/resources/metric_impact_source.md

```diff
@@ -47,20 +47,21 @@ resource "sleuth_metric_impact_source" "cloudwatch_rds_cpu" {
 
 ### Required
 
-- `environment_slug` (String) The environment slug
+- `environment_slug` (String) The slug of the environment that this metric impact source belongs to.
 - `name` (String) Impact source name
-- `project_slug` (String) The project for this environment
+- `project_slug` (String) The slug of the project that this metric impact source belongs to.
 - `provider_type` (String) Integration provider type
 - `query` (String) The metric query
 
 ### Optional
 
-- `integration_slug` (String) Integration slug is generated automatically when an integration is set up in Sleuth. By default, it matches the `provider_type`. Any value specified in the integration's `Description label` field gets appended to the `integration_slug`, spaces replaced with dashes, e.g. `cloudwatch-test`
+- `integration_slug` (String) The integration slug
```

Comment:
I guess we can omit internal implementation details we used to expose here...although it seems like that has been documented for a reason 🤔

Reply:
hm, it should remain, will check what caused the generate issue